README.md

```diff
@@ -950,6 +951,67 @@ terra app execute dsub \
 the `dsub` [README](https://github.com/DataBiosphere/dsub/blob/main/README.md#getting-started-on-google-cloud)
 .)
 
+#### Configuring Credentials for AWS Resources
```

Comment:
Re 1 and 3, yes...  this writes a fresh config by querying the server every time it's run and overwriting the old file.  No update-in-place.  My expectation is that this will be called in `.bashrc` in the notebook (inside an `eval`), and also in the notebook startup script (which runs on every startup in Sagemaker).  I will also probably add a bash alias in the notebooks to quickly refresh the config without having to specify arguments.

Re 2, great point!  I went ahead and added another directory level for the server name, so it ends up being `$HOME/.terra/.aws/SERVER_NAME/WORKSPACE_NAME.conf`.

This also makes me realize that we'll be scraping info from tags on the Sagemaker notebook instance in the startup script to figure out which environment/workspace we are in, so it will be helpful to attach a `cli_server` tag to the Sagemaker notebook instance when we get there.

Reply:
created https://verily.atlassian.net/browse/TERRA-546 for (1)